### ISSUE TITLE
Remember position doesn't work when app is left open

### ISSUE THREAD

**Issue by Morpog** [veskuh/Tweetian#54]
Remember position only works when I close the app. If I don't and new tweets arrive, I will be at top position instead of staying at last position.

**Comment by siteshwar** [comment]
@veskuh Not able to reproduce it (or am I missing something here?).

**Comment by siteshwar** [comment]
@veskuh Ok, I get it now. Basically our logic for marking tweets as read is flawed. If Tweetian is open and let's say 1 new tweet arrives, it will be automatically marked as read because it's visible on the screen (tweets which are visible on screen are automatically marked as read). This way even after multiple updates unread count will remain 0 and list will always remain on top.

**Comment by veskuh** [comment]
Yes, seems logical that it is related to visibility of items. Refreshing from cover and having more than screenful of new items seems to work.
